arelle/PackageManager.py

```diff
@@ -25,6 +27,8 @@
 TP_XSD = "http://www.xbrl.org/2016/taxonomy-package.xsd"
 CAT_XSD = "http://www.xbrl.org/2016/taxonomy-package-catalog.xsd"
 
+if TYPE_CHECKING:
+    from .Cntlr import Cntlr
```

Comment:
Also suggest full file path here

Reply:
Second this. We're trying to move away from relative imports.